### PR TITLE
tdigest v0.0.0-20181121200506-bf2b5ad3c0a9 build and validate script

### PR DESCRIPTION
**Pull request by ambujkumar3**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@prashantkhoje please review

File: t/tdigest/tdigest_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+# Maintainer    : Ambuj Kumar {Ambuj.Kumar3@ibm.com}
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#Run the script:./tdigest_ubi_8.5.sh v0.0.1(version_to_test)
```
Comment (prashantkhoje) [inline t/tdigest/tdigest_ubi_8.5.sh:20]: Please use format `# Comment` instead of `#Comment`

File: t/tdigest/tdigest_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,86 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : tdigest
+# Version       : v0.0.0-20181121200506-bf2b5ad3c0a9,v0.0.1
```
Comment (prashantkhoje) [inline t/tdigest/tdigest_ubi_8.5.sh:5]: Please change to `v0.0.0-20181121200506-bf2b5ad3c0a9, v0.0.1`

File: t/tdigest/tdigest_ubi_8.5.sh [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+rm -rf /home/tester/go && tar -C /home/tester -xzf go$GO_VERSION.linux-ppc64le.tar.gz
+rm -f go$GO_VERSION.linux-ppc64le.tar.gz
+export GOPATH=/home/tester/go
+export PATH=$PATH:$GOPATH/bin
+export  GO111MODULE=on
+mkdir -p $GOPATH/src/github.com/influxdata
+cd $GOPATH/src/github.com/influxdata
+rm -rf $PACKAGE_NAME
+
+
```
Comment (prashantkhoje) [inline t/tdigest/tdigest_ubi_8.5.sh:42]: Please remove extra blank line.

File: t/tdigest/tdigest_ubi_8.5.sh [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+
+
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+        echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
```
Comment (prashantkhoje) [inline t/tdigest/tdigest_ubi_8.5.sh:47]: Please use uniformed indentation in the file.

Comment (ambujkumar3) [reply t/tdigest/tdigest_ubi_8.5.sh:41]: Hi Prashant done the review changes

**Comment by prashantkhoje** [comment]
LGTM

**Pull request merged**